### ISSUE TITLE
[Feature request] Allow multiple accounts

### ISSUE THREAD

**Issue by PhiLhoSoft** [sindresorhus/anatine#18]
When I start Anatine with a window already opened, it just brings the existing window to front, and exits.
I would like to either be able to have two (or more) distinct instances opened on different accounts, or even better to be able to quickly switch from an account to another in the same window.

FYI, I use https://twitter.com/PhiLhoSoft for technical stuff, in English, and https://twitter.com/Phi_Lho for more personal stuff, following comic artists, making puns in French, etc. :smile: 
Currently, I have one account opened on Firefox, and the other on Chrome... (now on Anatine)

I imagine some people have personal account and manage professional ones, or similar needs.

**Comment by sindresorhus** [comment]
Yes, that would indeed be useful. I manage multiple accounts too, and been using [Twitcher](https://chrome.google.com/webstore/detail/twitcher-twitter-account/gmngpagflejjoblmmamaonmnkghjmebh).

A pull request would be appreciated as I doubt I will have time to implement this.
